dhis-2/dhis-services/dhis-service-validation/src/test/java/org/hisp/dhis/validation/ValidationResultStoreTest.java

```diff
@@ -195,8 +184,6 @@ private CurrentUserService getMockCurrentUserService( String userName, boolean s
 
     private void setPrivateAccess( BaseIdentifiableObject object, UserGroup... userGroups )
     {
-//        object.setPublicAccess( ACCESS_NONE );
```

Comment:
Remove commented code.

Reply:
Not sure what you mean - this removes the commended code as the 2 lines below appear to be the "new" way to do what was commented out here.

Reply:
Sorry!